tests/utils.go

```diff
@@ -1485,6 +1484,16 @@ func DeletePV(os string) {
 	}
 }
 
+func DeleteVMI(vmi *v1.VirtualMachineInstance) {
```

Comment:
nit: This func seems to fit the next commit.

Reply:
I wanted to keep the other commit just for the test.
I hope this function will bu used for other tests as well in the future so I kept it in the "general" commit.

Reply:
Ok, I will not insist on this but please note that I will not ack any additions to `utils`. Unfortunately one of the PR/s that should have helped with this is blocked on review, hope we can push it forward before I'll start blocking all other PR/s that are tempted to add more to utils.